Library/Formula/fonttools.rb

```diff
@@ -0,0 +1,27 @@
+class Fonttools < Formula
+  desc "FontTools is a library for manipulating fonts"
+  homepage "https://github.com/behdad/fonttools"
+  url "https://github.com/behdad/fonttools/archive/2.5.tar.gz"
+  sha256 "c89603f9f3346f48f4a24f786422e935423554e0a9172dcd3ec8ffbd556d2159"
+  head "https://github.com/behdad/fonttools.git"
+
+  option "with-pygtk", "Build with pygtk support for pyftinspect"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "pygtk" => :optional
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages/FontTools"
```

Comment:
@DomT4 
In the setup.py, the author use `extra_path = 'FontTools',` to put all core files in `site-packages/FontTools`, without this line, the program can't run at all.

But I found that I can remove line 14, this line is useless.